MMR.Randomizer/Utils/SequenceUtils.cs

```diff
@@ -777,6 +782,67 @@ private static void RelocateSeq(int f)
             ReadWriteUtils.WriteToROM(0x00C2739C, new byte[] { 0x3C, 0x08, 0x80, 0x0A, 0x8D, 0x05, (byte)(offset >> 8), (byte)(offset & 0xFF) });
         }
 
+        public static void MoveAudioBankTableToFile()
+        {
+            // grab original audiobanktable out of code, plus extra for modifying
+            var table = ReadWriteUtils.ReadBytes(0xB3C000 + 0x13B6C0, 0x820);
+            New_AudioBankTable = RomUtils.AddNewFile(table);
+
+            // instrumentset_patch: modifies audiobank metadata read and writes, instrument/drum/sfx pointer read and writes,
+            // nops a metadata copy function, and sets a fixed size for the audiobank pointer index
+            ResourceUtils.ApplyHack(Resources.mods.instrumentset_patch);
+
+            // moveaudiostatebytes: sets where read and writes for sequence and instrumentset states go
+            // in this hack, they're moved from 0x80205008 to end of old instrumentset table in code and given more space
+            // if these don't get moved, new banks at 0x30 and up will overflow into sequence states and can knock out sound
+            ResourceUtils.ApplyHack(Resources.mods.moveaudiostatebytes);
+
+            // loadnewaudiotable: where the copy metadata function loop was, sets a jump to code placed at the old instrumentset list
+            // which DMAs new audiobanktable from a file, relocates the addresses in the table, and sets the
+            // instrumentset table pointer to the new file
+            ResourceUtils.ApplyHack(Resources.mods.loadnewaudiotable);
```

Comment:
In this file, can you change `ADDIU SP, SP, 0x00` to `ADDIU SP, SP, -0x18` at the start, and + 0x18 at the end? And also `SW/LW RA 0x0010 (SP)` to `SW/LW RA 0x0014 (SP)`